nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1041,6 +1043,78 @@ async fn project_policy_update(
 
 // IP Pools
 
+/// List all IP Pools that can be used by a given project.
+#[endpoint {
+    method = GET,
+    path = "/v1/ip-pools",
+    tags = ["projects"],
+}]
+async fn ip_pool_list_project_scoped(
```

Comment:
```suggestion
async fn project_ip_pool_list(
```

To be consistent with our current naming scheme.